addons/l10n_si/data/account.account.type.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data noupdate="1">
+        <record id="account.data_account_type_income" model="account.account.type">
+            <field name="code">uspeh.prihodki</field>
+            <field name="note"></field>
+            <field name="name">Uspeh - prihodki</field>
+            <field name="close_method">none</field>
+            <field name="report_type">income</field>
+        </record>
```

Comment:
Thank you Stefan for pointing this out. We investigated why this was required and found out we can remove it.